h2o-core/src/main/java/water/rapids/ast/prims/mungers/AstMelt.java

```diff
@@ -0,0 +1,99 @@
+package water.rapids.ast.prims.mungers;
+
+import water.MRTask;
+import water.fvec.*;
+import water.rapids.Val;
+import water.rapids.ast.AstBuiltin;
+import water.rapids.vals.ValFrame;
+import water.util.ArrayUtils;
+
+public class AstMelt extends AstBuiltin<AstPivot> {
+
+  @Override
+  public String[] args() {
+    return new String[]{"frame", "id_vars", "value_vars", "var_name", "value_name", "skip_na"};
```

Comment:
Could be a constant, but hey, let's not keep it in memory. if it was a constant, it could be used in nargs. Ok, this is actually better.